client/app/scripts/superdesk-desks/aggregate.js

```diff
@@ -492,6 +520,14 @@
                     values = _.sortBy(values, function(item) {
                         return item.order;
                     });
+
+                    _.each(values, function(item) {
+                        if (item.type === 'deskOutput') {
+                            var desk_id = item._id.substring(0, item._id.indexOf(':'));
```

Comment:
It might make sense to refactor this logic of extracting content object's ID into a helper function, since identical lines appear at two other places in this file already.